samples/21.core-bot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/DialogAndWelcomeBot.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.corebot.app.insights;
+
+import com.codepoetics.protonpack.collectors.CompletableFutures;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.io.IOUtils;
+import com.microsoft.applicationinsights.core.dependencies.apachecommons.lang3.StringUtils;
```

Comment:
Is this ok? I mean, it's not ok to consume the dependencies of another dependency